source/manual/data-gov-uk-2nd-line.html.md

```diff
@@ -0,0 +1,84 @@
+
+#### Deleting a single dataset
+1. Login to [CKAN][ckan] as a sysadmin user (credentials are available in the `govuk-secrets` password store, under `datagovuk/ckan`).
+2. Navigate to the relevant dataset (use the 'Datasets' button in the header).
+3. Click the 'Manange' button.
+4. Click the red 'Delete' button.
+5. Once withdrawn, it will take up to 30 minutes to sync across to Find and clear the cache.
+
+#### Bulk deleting datasets
+Sometimes a publisher will request a large deletion of datasets.  In these cases, you can bulk withdraw the datasets using the instructions contained in the [supporting CKAN](/manual/data-gov-uk-supporting-ckan.html#deleting-a-dataset) documentation.  Refer to the instructions for 'Deleting a dataset' (which is a soft-delete).  **Do not purge the dataset - this is a hard-delete.**
+
+## Harvesting
+Harvesting is where publishers can automatically import their data to data.gov.uk without having to manually enter it into the web interface.  They can be set up to run automatically at specified periods, or run manually on-demand.
+
+### Get the status of a harvester
+1. Login to [CKAN][ckan] as a sysadmin user (credentials are available in the `govuk-secrets` password store, under `datagovuk/ckan`).
+2. Navigate to the relevant harvester (use the 'Harvest' button in the header).
+3. You will see a list of the datasets imported by this harvest source.
+4. Click the 'Manage' button to get the status.
+5. A summary of the current status will be shown.  Individual runs (and the error messages logged) can be access from the 'Jobs' tab.
+
+### Restart a harvester
+1. Login to [CKAN][ckan] as a sysadmin user (credentials are available in the `govuk-secrets` password store, under `datagovuk/ckan`).
+2. Navigate to the relevant harvester (use the 'Harvest' button in the header).
+3. You will see a list of the datasets imported by this harvest source.
+4. Click the 'Manage' button.
+5. If the harvester is currently running, click the 'Stop' button to stop it.  Once it has stopped, or if it is not currently running, click the 'Reharvest' button.  You will know if the harvester is running because the 'Reharvest' button will be disabled.
+
+## Organogram Publishing
+Organograms are files that allow the people structure of an organisation to be visualised.  They are split into two files: one for senior staff (grades SCS1, SCS2 and SCS3, or equivalent) and another for junior staff (all other grades).  The senior staff file is more detailed than the junior staff file, with staff names included for posts classified as grades SCS2 and SCS3.
+
+Organograms are unique in data.gov.uk since we actually host the data, rather than linking to an external resource.  The data files are hosted in a S3 bucket (named `datagovuk-integration-ckan-organogram`, `datagovuk-staging-ckan-organogram` or `datagovuk-production-ckan-organogram`, depending on the environment).
```

Comment:
When talking to devs about might be better to avoid using the word "unique"?
```suggestion
 Organograms are the only datasets in data.gov.uk where we host the data itself rather than linking to an external resource.  The data files are hosted in a S3 bucket (named `datagovuk-integration-ckan-organogram`, `datagovuk-staging-ckan-organogram` or `datagovuk-production-ckan-organogram`, depending on the environment).
```